.goreleaser.yml

```diff
@@ -26,7 +26,6 @@ archives:
   - format: binary
     replacements:
       386: i386
```

Comment:
How about this one, does it look right?

Reply:
I think this replacement shouldn't be there either, at least based on this: https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63.